ts/editable/mathjax-element.ts

```diff
@@ -5,6 +5,7 @@ import { on } from "@tslib/events";
 
 import { placeCaretAfter, placeCaretBefore } from "$lib/domlib/place-caret";
 
+import { createClassComponent } from "svelte/legacy";
```

Comment:
I used createClassComponent because $state() doesn't work here for some reason (tried adding a .svelte suffix to the filename)